masu/database/aws_report_db_accessor.py

```diff
@@ -92,6 +94,50 @@ def get_lineitem_query_for_billid(self, bill_id):
         line_item_query = base_query.filter(cost_entry_bill_id == bill_id)
         return line_item_query
 
+    def get_daily_query_for_billid(self, bill_id):
+        """Get the AWS cost daily item for a given bill query."""
+        table_name = AWS_CUR_TABLE_MAP['line_item_daily']
+        cost_entry_bill_id = getattr(
+            getattr(self.report_schema, table_name),
+            'cost_entry_bill_id'
+        )
+        base_query = self._get_db_obj_query(table_name)
+        daily_item_query = base_query.filter(cost_entry_bill_id == bill_id)
+        return daily_item_query
+
+    def get_summary_query_for_billid(self, bill_id):
+        """Get the AWS cost summary item for a given bill query."""
+        table_name = AWS_CUR_TABLE_MAP['line_item_daily_summary']
+        cost_entry_bill_id = getattr(
+            getattr(self.report_schema, table_name),
+            'cost_entry_bill_id'
+        )
+        base_query = self._get_db_obj_query(table_name)
+        summary_item_query = base_query.filter(cost_entry_bill_id == bill_id)
+        return summary_item_query
+
+    def get_ocp_aws_summary_query_for_billid(self, bill_id):
```

Comment:
I think we want this method and project_summary one below it to have an ocp equivalent that does it by cluster_id. In case you delete the OCP provider but not the AWS provider. The tables should be able to be cleared by cluster_id.